station/recipes/helpers.py

```diff
@@ -0,0 +1,31 @@
+from functools import wraps
+
+import sh
+
+
+def set_sh_defaults(f):
+    '''
+    Set default parameters in "sh";
```

Comment:
what is sh? A shell or a python module with the same name? Ok, figured that out by looking at some later changes. Anyway, there should be a comment, explaining what sh is.

Reply:
I extend the comment.